src/lib/components/FooterNav.svelte

```diff
@@ -46,7 +46,7 @@
                 },
                 {
                     label: 'Status',
-                    href: 'https://status.appwrite.io',
+                    href: 'https://www.appwrite.online',
```

Comment:
This leads to a page without a SSL certificate, and redirects to BetterStack's landing page